norfair/drawing/draw_boxes.py

```diff
@@ -182,6 +187,7 @@ def draw_tracked_boxes(
     draw_box: bool = True,
     color_by_label: bool = False,
     draw_labels: bool = False,
+    draw_scores: bool = False,
```

Comment:
No need to add this parameter here, this function is deprecated and is just here for backward compatibility